src/patternfly/components/InputGroup/examples/InputGroup.md

```diff
@@ -116,6 +116,21 @@ cssPrefix: pf-c-input-group
 {{/input-group}}
 ```
 
+### Width control
```

Comment:
I'm wondering if we need this for an input group, or if we could just enable it on a toolbar item? Was there a reason we need it on an input group, too?

Reply:
My reasoning was that the same issue would occur outside of a `toolbar`. But we can limit this PR to `toolbar`